whole_body_kinematic_controller:
  type: pal_wbc/WholeBodyControlKinematicController
  
  useFT: False
  useIMU: False
  dt: 0.005
  floating_base: FixedBase
  formulation: velocity
  stack_configuration: tiago_fixed_base_stack
  solver_type: QpReductionuEqualitiesQuadprogHeapAllocation

  use_home_posture: True
  
  robot_model_chains:
    [wrist_ft_link,
     xtion_optical_frame]

  default_configuration:
    arm_1_joint: 1.7
    arm_2_joint: -0.5
    arm_3_joint: -3.4
    arm_4_joint: 0.5
    arm_5_joint: 0.0
    arm_6_joint: 0.0
    arm_7_joint: 0.0
    head_1_joint: 0.0 
    head_2_joint: 0.0
    torso_lift_joint: 0.14

  position_joints:
    - arm_1_joint
    - arm_2_joint
    - arm_3_joint
    - arm_4_joint
    - arm_5_joint
    - arm_6_joint
    - arm_7_joint
    - head_1_joint
    - head_2_joint
    - torso_lift_joint